WasatchNET/EEPROM.cs

```diff
@@ -870,11 +870,13 @@ public bool write()
                 if (!ParseData.writeString(productConfiguration, pages[5], 30, 16)) return false;
 
                 if (!ParseData.writeByte(intensityCorrectionOrder, pages[6], 0)) return false;
-                for (int i = 0; i < intensityCorrectionCoeffs.Length; ++i)
+                if (intensityCorrectionOrder < 8)

```

Comment:
In retrospect (this isn't a ding on you), once we decided that all SRM calibrations were to be polynomial, and once I decided to allocate 8 coeffs for a 7th-order polynomial, we should have eliminated that "order" byte altogether (just leave any unneeded coeffs zero).  There really is no need for that byte to exist.  I'm fine leaving it until we move to the bigger EEPROM in the NextGen boards and we do major cleanup on the field layout.